R/OptimInstance.R

```diff
@@ -81,8 +89,10 @@ OptimInstance = R6Class("OptimInstance",
         stop(terminated_error(self))
       }
       xss_trafoed = transform_xdt_to_xss(xdt, self$search_space)
+      private$.log_eval_batch_start(xdt)
       ydt = self$objective$eval_many(xss_trafoed)
       self$archive$add_evals(xdt, xss_trafoed, ydt)
+      private$.log_eval_batch_finish()
```

Comment:
I think that's a good idea. The archive subset does not have any more information since `batch_nr` is printed in the line above and `opt_x` is always a list.